spec/mailer_previews/user_preview.rb

```diff
@@ -0,0 +1,42 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
+
+  def email_confirmation
+    show_email do
+      @mail = UserMailer.email_confirmation(@user).message
```

Comment:
you can make the user unconfirmed here in the show block instead of commenting the if in the mailer that is so amateurish...